### ISSUE TITLE
a11y: Fix problem on CI a11y test where report is not being generated properly

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#15636]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

Fix problem on CI a11y test where report is not being generated properly

**To Reproduce**

Check the output of the `Upload Report` step in our `a11y-test` CI job, like https://github.com/rancher/dashboard/actions/runs/18472125732/job/52628444522?pr=15509

If you download the artifacts from that link there's no HTML report generated, nor the associated json with the report information.

**Result**
There's no HTML report generated, nor the associated json

**Expected Result**

We should have a HTML report generated

**Screenshots**

**Additional context**

**Comment by a-arias** [comment]
Requiere manual validation

**Comment by a-arias** [comment]
**To Test**
Fix validated ✅

Rancher :v2.13-head
UI: v2.13 commit a14f19abd61f70f40d77ce5c6ae752d5d6449a04

<img width="1531" height="915" alt="Image" src="https://github.com/user-attachments/assets/1ce7b698-d2e8-4d6c-88aa-ecef986d2c32" />

<img width="950" height="487" alt="Image" src="https://github.com/user-attachments/assets/8516a982-7213-4db1-8108-e55aefbc6846" />
